modules/adding-yaml-examples-to-kube-resources.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// * web-console/customizing-the-web-console.adoc
+
+[id="adding-yaml-examples-to-kube-resources_{context}"]
+= Adding YAML examples to Kubernetes resources
+
+You can dynamically add YAML examples to any Kubernetes resources at any time.
+
+.Procedure
+
+. From *Administration* -> *Custom Resource Definitions*, click on *ConsoleYAMLSample*.
+
+. Click the *Create Console Instances Sample* button.
```

Comment:
I don't see this button. We might suggest using Home -> Search for this since the Custom Resource Definitions list will be hidden from normal users.

Home -> Search -> ConsoleYAMLSample -> Create Console YAML Sample

Do we recommend creating the other extension resources the same way? We should be consistent.

It's probably worth calling out that typically only admins can create these resources (for YAML sample and other extension custom resources)